web/modules/custom/dbcdk_community/src/Plugin/Block/FlaggedContentDetails.php

```diff
@@ -0,0 +1,189 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\dbcdk_community\Plugin\Block\FlaggedContentDetails.
+ */
+
+namespace Drupal\dbcdk_community\Plugin\Block;
+
+use DBCDK\CommunityServices\Api\ProfileApi;
+use DBCDK\CommunityServices\ApiException;
+use Drupal\Core\Block\BlockBase;
+use Drupal\Core\Link;
+use Drupal\Core\Plugin\ContainerFactoryPluginInterface;
+use Drupal\Core\Url;
+use Drupal\dbcdk_community\Url\ModelUrlGenerator;
+use Drupal\dbcdk_community\Url\PropertyUrlGenerator;
+use Drupal\dbcdk_community\Url\UrlGeneratorInterface;
+use Psr\Log\LoggerInterface;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Drupal\dbcdk_community\Content\FlaggableContentRepository;
+
+/**
+ * Provides a 'FlaggedContentDetails' block.
+ *
+ * @Block(
+ *  id = "flagged_content_details",
+ *  admin_label = @Translation("Flagged content details"),
+ *  context = {
+ *    "flag_id" = @ContextDefinition("integer")
+ *  }
+ * )
+ */
+class FlaggedContentDetails extends BlockBase implements ContainerFactoryPluginInterface {
+
+  /**
+   * The repository to use when retrieving flaggable content.
+   *
+   * @var FlaggableContentRepository
+   */
+  protected $flaggableContentRepository;
+
+  /**
+   * The profile api to use when retrieving information about content owners.
+   *
+   * @var ProfileApi
+   */
+  protected $profileApi;
+
+  /**
+   * The url generator to use when creating links to the biblo.dk site.
+   *
+   * @var UrlGeneratorInterface
+   */
+  protected $urlGenerator;
+
+  /**
+   * The logger to use.
+   *
+   * @var LoggerInterface
+   */
+  protected $logger;
+
+  /**
+   * Construct.
+   *
+   * @param array $configuration
+   *   A configuration array containing information about the plugin instance.
+   * @param string $plugin_id
+   *   The plugin_id for the plugin instance.
+   * @param string $plugin_definition
+   *   The plugin implementation definition.
+   * @param LoggerInterface $logger
+   *   The logger to use.
+   * @param FlaggableContentRepository $repository
+   *   The repository to use when retrieving flaggable content.
+   * @param ProfileApi $profile_api
+   *   The profile api to use when retrieving information about content owners.
+   * @param UrlGeneratorInterface $url_generator
+   *   The url generator to use when generating links to the biblo.dk site.
+   */
+  public function __construct(
+        array $configuration,
+        $plugin_id,
+        $plugin_definition,
+        LoggerInterface $logger,
+        FlaggableContentRepository $repository,
+        ProfileApi $profile_api,
+        UrlGeneratorInterface $url_generator
```

Comment:
Does Code Sniffer approve this? It seems like it would expect 2 less spaces in the indentation?